bootstrap.yml

```diff
@@ -14,7 +14,7 @@
       changed_when:
         - output.stdout != ""
         - output.stdout != "\r\n"
-      when: ansible_os_family == "Debian"
+      when: islandora_vagrant_distro == "ubuntu/xenial64"
```

Comment:
To keep the play from failing when this isn't defined. Say not run with vagrant, I would do something like: 
```yaml 
when: islandora_vagrant_distro|default('false') == "ubuntu/xenial64"
```

So there is a default value. And don't install python with apt by default.

Reply:
I would suggest that we provide a default value for `islandora_vagrant_distro` variable such as `ubuntu/xenial64`.

Also a more generic variable name is more appropriate, `islandora_distro`.